internal_transfers/actions/src/utils.ts

```diff
@@ -22,7 +22,12 @@ export async function getTxDataFromHash(
   const { from, hash, blockNumber, logs: readonlyLogs } = transaction;
   const logs = readonlyLogs.map((log) => {
     return {
-      ...log,
+      // Prune TxReceipt:
+      // could use `...log` here, but that comes with undesired bulk
+      // e.g. https://jsonblob.com/1106016587013701632
+      index: log.index,
+      address: log.address,
+      data: log.data,
```

Comment:
Completely aesthetics choice IMO, but if you wanted, you could de-structure `log` in the function params, also get rid of the `return` statement.

Makes no difference in the end, just pointing it out

Reply:
I would be happy to try this, but not sure what you mean here...

Reply:
Got it thanks!